utils/travis/publish-gh-pages.js

```diff
@@ -11,5 +11,6 @@ const ghPages = require('gh-pages');
 
 ghPages.publish('demo', {
   repo: `https://${process.env.GITHUB_TOKEN}@github.com/zendeskgarden/react-components.git`,
-  silent: true
+  silent: true,
+  add: true
```

Comment:
Are we sure this is the right thing to do? I just don't want to end up with transient content that should be taken down, but never is.

Reply:
From our offline talk. This is the only reliable way for us to pre-publish PRs for review without stepping on each others toes. A better solution for the future is having a PR based staging site, but until then, use `add`.